bucket.go

```diff
@@ -28,6 +30,7 @@ type Bucket interface {
 	SetRaw(k string, exp int, v []byte) error
 	Delete(k string) error
 	Write(k string, flags int, exp int, v interface{}, opt WriteOptions) error
+	WriteCas(k string, flags int, exp int, cas uint64, v interface{}, opt WriteOptions) (ocas uint64, err error)
```

Comment:
It's just to differentiate from the input 'cas' parameter - it's meant to be shorthand for 'cas out'.  If there's a better naming convention that comes to mind, let me know.